### ISSUE TITLE
Manga Reader: vertical reading mode

### ISSUE THREAD

**Issue by LuceusXylian** [proxer/ProxerAndroid#34]
I miss the vertical reading mode. I liked it very much!

With the vertical reading mode you could simply scroll through the mangas like on [webtoons.com](https://webtoons.com).

**Comment by rubengees** [comment]
The app still has a vertical mode. You can toggle it via the button in the top right. Does that not work for you?

**Comment by XylianZeref** [comment]
To get the menu where the toggle button is, the user has to touch an image. My mistake was that I was trying to get this menu on the chapter controll page / 0 page.

For a improvment I think we should always show the title bar with the buttons on page 0 and on the next page it vanish.

**Comment by rubengees** [comment]
I agree! Actually I once tried to implement that, but could not get that working very good...
